packages/terra-consumer-nav/src/components/user-profile/ProfileLinks.jsx

```diff
@@ -1,42 +1,88 @@
 import React from 'react';
```

Comment:
dang, this seems very similar to our other nav link toggle components with children. Is there any re-usability, or are they just styled very differently?

Reply:
i was thinking that too, I want to look into extracting this into a single reusable component but for tims'e sake i went for functional first